plans/splitbrain/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/testground/sdk-go/network"
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+)
+
+type brainside int
+
+const (
+	leftbrain = iota
+	rightbrain
+	galaxybrain
+)
+
+func (s brainside) String() string {
+	return [...]string{"leftbrain", "rightbrain", "galaxybrain"}[s]
+}
+
+type node struct {
+	Side  int
+	IPNet *net.IPNet
+}
+
+func main() {
+	testcases := map[string]runtime.TestCaseFn{
+		"drop":   shirtsAndSkins(network.Drop),
+		"reject": shirtsAndSkins(network.Reject),
+		"accept": shirtsAndSkins(network.Accept),
+	}
+	runtime.InvokeMap(testcases)
+}
+
+func setup(ctx context.Context, runenv *runtime.RunEnv) (client *sync.Client, nwclient *network.Client) {
+	client = sync.MustBoundClient(ctx, runenv)
+	defer client.Close()
+
+	if !runenv.TestSidecar {
+		return nil, nil
+	}
+
+	nwclient = network.NewClient(client, runenv)
```

Comment:
Convention is to call it `netclient`.

```suggestion
	netclient = network.NewClient(client, runenv)
```